.vsts-ci/misc-analysis.yml

```diff
@@ -33,24 +33,22 @@ phases:
     displayName: Capture environment
     condition: succeededOrFailed()
 
-  - powershell:  Install-module pester -Scope CurrentUser -Force -SkipPublisherCheck
+  - powershell:  Install-Module -Name pester -Scope CurrentUser -Force
     displayName: Install Pester
     condition: succeededOrFailed()
 
-  - bash: |
-      sudo npm install -g markdown-spellcheck@0.11.0
-      # Sometimes this folder is left behind with root permissions and is needed by later NPM installs which don't need sudo
-      sudo rm -rf ~/.npm/_cacache
-    displayName: Install mdspell
+  - powershell: Install-Module -Name ThreadJob -Scope CurrentUser -Force
+    displayName: Install ThreadJob
     condition: succeededOrFailed()
 
   - powershell:  Write-Host "##vso[build.updatebuildnumber]$env:BUILD_SOURCEBRANCHNAME-$env:BUILD_SOURCEVERSION-$((get-date).ToString("yyyyMMddhhmmss"))"
     displayName: Set Build Name for Non-PR
     condition: ne(variables['Build.Reason'], 'PullRequest')
 
+  # Sometimes this folder is left behind with root permissions and causes many problems
   - bash: |
-      mdspell '**/*.md' '!**/Pester/**/*.md' --ignore-numbers --ignore-acronyms --report --en-us;
-    displayName: Test Spelling
+      sudo rm -rf ~/.npm/_cacache
+    displayName: Remove Old NPM Caches
```

Comment:
Revert style.

Reply:
This isn't style, had to do it to make it a pester test...